Source/WebCore/rendering/cocoa/RenderThemeCocoa.mm

```diff
@@ -2649,6 +2655,13 @@ static void adjustInputElementButtonStyleForVectorBasedControls(RenderStyle& sty
         style.setColor(buttonTextColor(styleColorOptions, !element->isDisabledFormControl()));
     }
 
+    // Enforce some default styles in the case that this is a non-multiple <select> element
+    // on any Cocoa platform, or a date input on iOS. We don't force these if this is just
+    // an element with "-webkit-appearance: menulist-button".
```

Comment:
I think this comment can be removed too. 

> We don't force these if this is just an element with "-webkit-appearance: menulist-button"

Is particularly unnecessary now that we no longer allow arbitrary HTML elements to have `appearance`.